tests/lib/snaps/test-snapd-content-slot-empty-content-attr/meta/snap.yaml

```diff
@@ -0,0 +1,7 @@
+name: test-snapd-content-slot-empty-content-attr
+version: 1.0
+slots:
+    shared-content:
+        interface: content
+        read:
+            - /
```

Comment:
This should say `$SNAP/` instead. The old syntax is legacy and we should not advertise it more.